integration-cli/docker_cli_ps_test.go

```diff
@@ -804,3 +804,60 @@ func (s *DockerSuite) TestPsFormatSize(c *check.C) {
 	lines = strings.Split(out, "\n")
 	c.Assert(lines[8], checker.HasPrefix, "size:", check.Commentf("Size should be appended on a newline"))
 }
+
+func (s *DockerSuite) TestPsListContainersFilterNetwork(c *check.C) {
+	// create a network, and some containers
```

Comment:
Or make it conditional, and filter on "nat" for windows. Didn't come round to do this yet, will try to do so later today